testutils/moby-defaults.go

```diff
@@ -76,3 +77,27 @@ func getDefaultCapSet() map[types.Capability]bool {
 
 	return capSet
 }
+
+// GetNonDefaultMounts returns a mount set from the provided mount list without default Moby mounts that it may contain
+func GetNonDefaultMounts(mountList []specs.Mount) []specs.Mount {
```

Comment:
Can we add a test with this with a non-empty mountList?  It seems like it isn't being tested from the coverage report

Reply:
done!